sdk/inc/azure/core/internal/az_http_internal.h

```diff
@@ -83,9 +83,9 @@ AZ_NODISCARD AZ_INLINE _az_http_policy_apiversion_options
 _az_http_policy_apiversion_options_default()
 {
   return (_az_http_policy_apiversion_options){
-    ._internal = { .option_location = _az_http_policy_apiversion_option_location_header,
-                   .name = AZ_SPAN_EMPTY,
-                   .version = AZ_SPAN_EMPTY }
+    ._internal = { .name = AZ_SPAN_EMPTY,
```

Comment:
For reference, here's the order in the struct:
https://github.com/Azure/azure-sdk-for-c/blob/5ad7985d50af8873c6148f867f57c487e294fb28/sdk/inc/azure/core/internal/az_http_internal.h#L42-L56